package.json

```diff
@@ -106,6 +106,20 @@
                                 "type": "number",
                                 "description": "A port to be used to enable automatic reloading of breakpoints when sourcemaps change.",
                                 "default": 9090
+                            },
+                            "sourceMaps": {
+                                "type": "boolean",
+                                "description": "Whether to use JavaScript source maps to map the generated bundled code back to it's original sources",
+                                "default": false
+                            },
+                            "logCatArguments": {
+                                "type": "array",
+                                "description": "Arguments to be used for LogCat (The LogCat output will appear on an Output Channel). It can be either such as an array: [\":S\", \"ReactNative:V\", \"ReactNativeJS:V\"] or a string such as \":S ReactNative:V ReactNativeJS:V\"",
```

Comment:
Minor typo / reword for digeff's description: `It can be either such as an array` -> `It can either be an array such as`